lib/win32/windows/constants.rb

```diff
@@ -60,19 +60,19 @@ module TaskSchedulerConstants
     # Weekly tasks
 
     # The task will run between the 1st and 7th day of the month
-    TASK_FIRST_WEEK = 1
+    TASK_FIRST_WEEK = 0X01
```

Comment:
you probably want all these to be `0x01` instead of `0X01` for style. I don't think it functionally matters.

Reply:
Yes I follow for style also but also there is change in the values as per MSDN library 
setting third, fourth, last was not working updated constant values by referring to
https://msdn.microsoft.com/en-us/library/windows/desktop/aa380733(v=vs.85).aspx

Reply:
I'm on my phone, so I can't dig into it, but I think that page is just wrong. In particular ruby is going to return the same value.

Hop into `irb` and type `0X01` and then `0x01` and confirm you get the same values. Ruby is going to interpret the 0X as hex but when reading code it's a lot easier to see that it's a hex value when scanning.

Reply:
@btm yes the value on irb for both are same.